ui/pages/groups/index.js

```diff
@@ -57,6 +57,11 @@ function AddGroupsDialog({ setOpen }) {
                 autoFocus
                 spellCheck='false'
                 type='search'
+                onKeyDown={e => {
+                  if (e.key === 'Escape') {
```

Comment:
nit:
In IE/Edge this may be 'Esc' rather than 'Escape'
```suggestion
                  if (e.key === 'Escape' || e.key === 'Esc') {
```

Reply:
oh i see. what if we compare the `keyCode` instead - the keycode is the same through out all the browsers

Reply:
I think that would work too

Reply:
oh nvm .. keycode is actually deprecated.